app/src/main/shared/reducers/authReducer.ts

```diff
@@ -40,9 +56,18 @@ export function loadAuthState(options: AuthStateOptions): AuthState {
         isAccountActive: permissionsInclude(options.permissions,"*/can-login"),
         isModeller: options.modellingGroups.length > 0,
         username: options.username,
-        permissions: options.permissions,
         modellingGroups: options.modellingGroups,
-        canUploadCoverage: permissionsInclude(options.permissions, "*/coverage.write")
+
+        canDownloadCoverage: permissionsInclude(options.permissions,"*/coverage.read"),
+        canUploadCoverage: permissionsInclude(options.permissions, "*/coverage.write"),
+        canViewGroups: permissionsInclude(options.permissions,"*/modelling-groups.read"),
+        canViewTouchstones: permissionsInclude(options.permissions,"*/touchstones.read"),
+        canViewUsers: permissionsInclude(options.permissions, "*/users.read"),
+        canReadRoles: permissionsInclude(options.permissions,"*/roles.read"),
+        canWriteRoles: permissionsInclude(options.permissions,"*/roles.write"),
+        canCreateUsers: permissionsInclude(options.permissions,"*/users.create"),
+        canCreateModellingGroups: permissionsInclude(options.permissions,"*/users.create"),
```

Comment:
We're actually using the same permission for this as for canCreateUser, but the components don't need to know this.

Reply:
Can you provide a bit more context for the apparent anomaly of `modelling-groups.read` vs `users.create`?